packages/shared/src/components/UserBadge.tsx

```diff
@@ -0,0 +1,29 @@
+import classNames from 'classnames';
+import React, { ReactElement, ReactNode } from 'react';
+import { IconProps } from './Icon';
+
+export type UserBadgeProps = {
+  className?: string;
+  content: ReactNode;
+  icon: ReactElement<IconProps>;
+};
+
+const UserBadge = ({
+  className,
+  content,
+  icon,
+}: UserBadgeProps): ReactElement => {
+  return (
+    <span
+      className={classNames(
+        'flex items-center ml-2 typo-footnote font-bold capitalize',
```

Comment:
I think there is always a gap between the icon and the label itself, not sure of the exact spacing, but I think it can be a good idea to support it OOTB.

```suggestion
        'flex gap-2 items-center ml-2 typo-footnote font-bold capitalize',
```